vita3k/gui/src/game_context_menu.cpp

```diff
@@ -52,9 +63,15 @@ void delete_game(GuiState &gui, HostState &host, const std::string &title_id) {
     }
 }
 
+static auto delete_game_popup = false;
+static auto delete_savedata_popup = false;
+static auto check_savedata_and_shaderlog = false;
+static auto copy_save_data_popup = false;
```

Comment:
:/

Reply:
I have changed nothing fir it, i just move place butbis same